packages/action-button/src/action-button.css

```diff
@@ -25,7 +25,7 @@ governing permissions and limitations under the License.
     -webkit-appearance: none;
 }
 
-:host(.spectrum-Dropdown-trigger) #button {
+:host(.spectrum-Picker-trigger) #button {
```

Comment:
At some point this whole rule when out of date and is no longer needed. Please remove.